### PR TITLE
Update README.md - first attempt

### PR DESCRIPTION
**Pull request by seemcat**
Made updates based on experience with developers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,40 +33,142 @@ Clover Go supports two types of card readers: a magnetic stripe, EMV chip-and-si
...
+        PARAMETERS.accessToken = "Access token that you generated via the OAuth flow earlier"
+        PARAMETERS.apiKey = "Get this value from your DevRel representative"
+        PARAMETERS.secret = "Get this value from your DevRel representative"
+        if let savedEndpoint = UserDefaults.standard.string(forKey: WS_ENDPOINT) {
+            endpointTextField.text = savedEndpoint
+        }
+    }
+```
+
+### FAQ
```
Comment (seemcat) [inline README.md]: FAQ's?

Comment (seemcat) [reply README.md]: Also, should this be **## FAQ's** (a bigger title)?

Comment (ianlsveto) [reply README.md]: should be ok

**Review by ianlsveto — APPROVED** [review]
lgtm

File: README.md [hunk 2]
```diff
@@ -33,40 +33,142 @@ Clover Go supports two types of card readers: a magnetic stripe, EMV chip-and-si
...
+
+#### Prerequisites
+1. Clone the SDK, go into the Example folder and check out the branch for Swift 5:
+
+       git clone https://github.com/clover/remote-pay-ios-go.git
+       cd remote-pay-ios-go/Example
+       git checkout CloverGo_Swift5.0
+2. Ensure that the following dependencies in your `Podfile` are set to the right versions:
+
+       pod 'Starscream', :git => 'https://github.com/daltoniam/Starscream.git', :tag => '3.0.5'pod 'GoConnector', '3.3.5'
```
Comment (madhavsahuclover) [inline README.md]: if its 3.3.5 we should be asking them to use Xcode 11.1 and set Swift 5 as discussed

Comment (seemcat) [reply README.md]: Have made this more explicit.

**Comment by frankfaustino** [comment]
👍LGTM! (Omit the `images/.DS_Store` file from git though)

File: README.md [hunk 3]
```diff
@@ -33,40 +33,143 @@ Clover Go supports two types of card readers: a magnetic stripe, EMV chip-and-si
...
+   Please make sure that your Clover Go reader is on and that your Android or Apple device has bluetooth on.
+- **I’ve tried everything and my app is still running into issues when attempting a transaction in Prod.**
+
+   Please check if you are using a Production reader by ensuring that there is no "Development" text on your device. A sandbox reader will have the word "Development" on the device, while a production reader will not.
+   
+   If you have the correct reader, please make sure your app has disabled Customer R/W, Employees W and enabled the rest of the Permissions. Btw, if you've recently changed your app’s Permissions settings, you will need to uninstall and reinstall the app, and re-generate the access token. This is because earlier tokens you have will only work for older requested permissions.
+
+   If our suggestions above do not work, we strongly encourage you to use your sandbox API key and secret to experiment with our sample app, to ensure that you understand how to accomplish certain implementations.
+
 ## Developer XCode iOS Project Setup
```
Comment (seemcat) [inline README.md]: Remove

**Review by madhavsahuclover — APPROVED** [review]

**Pull request merged**